modules/data-table/test/spec.coffee

```diff
@@ -465,7 +467,7 @@ describe 'data-table', ->
               done()
 
             container
-              .select('.hx-data-table-pagination-picker')
+              .select('.hx-data-table-paginator-picker')
```

Comment:
changed everywhere we use 'pagination' to 'paginator' because it was confusing having the two different variations for similar components